packages/esm-patient-attachments-app/src/attachments/attachments-overview.scss

```
@@ -23,7 +23,7 @@
 .divider {
   width: 1px;
   height: layout.$spacing-05;
-  color: colors.$gray-20;
+  color: $ui-05;
```

Looks like the cause of the build failures:
```suggestion
  color: vars.$ui-05;
```

---

Corrected, thank you @NethmiRodrigo .